src/constants/translations/ua/common.json

```diff
@@ -42,23 +45,52 @@
   "noItems": "Елементів не знайдено",
   "view": "Переглянути",
   "labels": {
+    "email": "Електронна пошта",
+    "firstName": "Ім'я",
+    "lastName": "Прізвище",
+    "password": "Пароль",
+    "confirmPassword": "Підтвердити пароль",
+    "terms": "Умови",
+    "privacyPolicy": "Політика конфіденційності",
+    "login": "Вхід",
+    "signup": "Реєстрація",
+    "country": "Країна",
+    "city": "Місто",
     "sendMessage": "Надіслати повідомлення",
     "viewDetails": "Переглянути деталі",
-    "sortBy": "Сортувати за:",
-    "subject": "Предмет:",
-    "level": "Рівень:",
+    "enrollOffer": "Відгукнутись на пропозицію",
     "makeActive": "Зробити активним",
     "moveToDraft": "Перемістити до чернетки",
-    "closeOffer": "Закрити пропозицію"
+    "closeOffer": "Закрити пропозицію",
+    "sortBy": "Сортувати за:",
+    "subject": "Предмет:",
+    "level": "Рівень:"
   },
   "errorMessages": {
+    "emailValid": "Електронна пошта має бути такого формату: “local-part@domain.com”",
+    "passwordLength": "Пароль не може бути коротшим за 8 і довшим за 25 символів",
+    "passwordAlphabeticAndNumeric": "Пароль повинен містити принаймні одну літеру та одну цифру",
+    "passwordValidSymbols": "Пароль повинен містити лише букви, цифри та спеціальні символи",
+    "emptyField": "Це поле не може бути порожнім",
+    "nameLength": "Довжина цього поля не може перевищувати 30 символів",
+    "nameAlphabeticOnly": "Це поле може містити лише букви",
+    "passwordsDontMatch": "Паролі не збігаються",
+    "shortText": "Цей текст занадто короткий",
+    "longText": "Цей текст занадто довгий",
+    "numbersOnly": "Введіть дійсний номер",
+    "positiveNumbersOnly": "Введіть позитивне число",
+    "title": "Будь ласка, укажіть назву",
+    "description": "Будь ласка, надайте опис",
+    "category": "Будь ласка, виберіть категорію",
+    "subject": "Будь ласка, виберіть предмет",
+    "proficiencyLevel": "Будь ласка, виберіть принаймні один рівень знань",
     "youtubeLink": "Будь ласка, надайте посилання на YouTube відео"
   },
   "levels": {
     "beginner": "Початковий",
     "intermediate": "Середній",
     "advanced": "Високий",
-    "testPreparation": "Підготовка до тесту",
+    "test preparation": "Підготовка до тесту",
```

Comment:
Why do we use space here? I think we should stick to `camel case` convention

Reply:
fixed, but a bug was discovered) will create an issue for it

Reply:
Sure